wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java

```diff
@@ -192,11 +192,8 @@ public static boolean isFailedState(String state) throws Exception
             WuStateNameMap.put("RUNNING", WUState.RUNNING);
     }
 
-    public static WUState translateWUState(String state) throws Exception
+    public static WUState translateWUState(String state)
     {
-        if (WuStateNameMap.size() <= 0)
```

Comment:
I didn't think it was worth throwing an exception to catch a static block not executing, that's defined right above.  I would have then had to handle the exception by swallowing, returning UNKNOWN, or changing the sig to also throw.